deeppavlov/utils/alice/alice.py

```diff
@@ -56,126 +54,69 @@ def interact_alice(agent: Agent):
         },
         'version': '1.0'
     }
 
-    agent_response: Union[str, RichMessage] = agent([payload or text], [dialog_id])[0]
-    if isinstance(agent_response, RichMessage):
-        response['response']['text'] = '\n'.join([j['content']
-                                                  for j in agent_response.json()
-                                                  if j['type'] == 'plain_text'])
-    else:
-        response['response']['text'] = str(agent_response)
-
-    return jsonify(response), 200
+    return response
 
 
-def start_alice_server(model_config, https=False, ssl_key=None, ssl_cert=None, port=None):
+def start_alice_server(model_config: Path,
+                       https: bool = False,
+                       ssl_key: Optional[str] = None,
+                       ssl_cert: Optional[str] = None,
+                       port: Optional[int] = None) -> None:
     server_config_path = get_settings_path() / SERVER_CONFIG_FILENAME
     server_params = get_server_params(server_config_path, model_config)
 
     https = https or server_params['https']
-    docs_endpoint = server_params['docs_endpoint']
-
-    Swagger.DEFAULT_CONFIG['specs_route'] = docs_endpoint
-    Swagger(app)
-
-    if not https:
-        ssl_key = ssl_cert = None
-    else:
-        ssh_key = Path(ssl_key or server_params['https_key_path']).resolve()
-        if not ssh_key.is_file():
-            e = FileNotFoundError('Ssh key file not found: please provide correct path in --key param or '
-                                  'https_key_path param in server configuration file')
-            log.error(e)
-            raise e
-
-        ssh_cert = Path(ssl_cert or server_params['https_cert_path']).resolve()
-        if not ssh_cert.is_file():
-            e = FileNotFoundError('Ssh certificate file not found: please provide correct path in --cert param or '
-                                  'https_cert_path param in server configuration file')
-            log.error(e)
-            raise e
-
     host = server_params['host']
     port = port or server_params['port']
     model_endpoint = server_params['model_endpoint']
 
+    ssl_config = get_ssl_params(server_params, https, ssl_key=ssl_key, ssl_cert=ssl_cert)
+
     model = build_model(model_config)
     skill = DefaultStatelessSkill(model, lang='ru')
     agent = DefaultAgent([skill], skills_processor=DefaultRichContentWrapper())
 
-    @app.route('/')
-    def index():
-        return redirect(docs_endpoint)
+    start_agent_server(agent, host, port, model_endpoint, ssl_config=ssl_config)
 
-    start_agent_server(agent, host, port, model_endpoint, ssl_key, ssl_cert)
 
+def start_agent_server(agent: Agent,
+                       host: str,
+                       port: int,
+                       endpoint: str,
+                       ssl_key: Optional[str] = None,
+                       ssl_cert: Optional[str] = None,
+                       ssl_config: Optional[SSLConfig] = None) -> None:
```

Comment:
Get rid of ambiguity: ((`ssl_key` and `ssl_cert`) **xor** `ssl_config`) **or** none of these args should be passed to this method. Add check with explicit exception raise when all these arguments are not none.